openfisca_us/variables/finance/benefit/inputs.py

```diff
@@ -0,0 +1,24 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+
+
+class gross_income(Variable):
+    value_type = float
+    entity = BenefitUnit
+    label = u"Income before SNAP program deductions are applied"
+    definition_period = YEAR
+
+
+class net_income(Variable):
```

Comment:
Is this definitely an input variable? Will it ever have a formula?

Reply:
It'll probably be a formula

Reply:
rename to spm_gross_income and move to finance/benefits/outputs.py